src/models/v2/custom-converters/gui-attribute-converter.ts

```
@@ -0,0 +1,21 @@
+import { JsonConverter, JsonCustomConvert } from "json2typescript";
+
+@JsonConverter
+export class GuiAttributeConverter implements JsonCustomConvert<string[]> {
+
+    serialize(attrs: string[]): any {
+    }
```

Should we throw exceptions for the missing serializations? I know there are some more, just asking in general.

---

I think we should think about how to create or modify ontologies / resources / values as soon as possible. In these cases, we would need the `serialize` method.